packages/react-components/react-spinner/library/src/components/Spinner/Spinner.test.tsx

```diff
@@ -45,4 +45,9 @@ describe('Spinner', () => {
     expect(result.container.getElementsByClassName('fui-Spinner__Progressbar')).toBeNull;
     expect(result.container.getElementsByClassName('fui-Spinner__label')).toBeNull;
   });
+
+  it('renders span as a root slot tag', () => {
+    const result = render(<Spinner as="span" />);
+    expect(result.queryByRole('progressbar')?.tagName).toBe('SPAN');
```

Comment:
won't it be more correct to have `getByRole` here? (and the whole test file). The spinner is always present. 

```suggestion
    expect(result.getByRole('progressbar')?.tagName).toBe('SPAN');
```